application/frontend/src/providers/DataProvider.tsx

```diff
@@ -38,12 +39,28 @@ export const DataProvider = ({ children }: { children: React.ReactNode }) => {
     keyPath.push(selfKey);
 
     const storedDoc = structuredClone(dataStore[selfKey]);
-    storedDoc.links = storedDoc.links
-      .filter(
-        (x) =>
-          x.document && !keyPath.includes(getStoreKey(x.document)) && getStoreKey(x.document) in dataStore
-      )
-      .map((x) => ({ ltype: x.ltype, document: buildTree(x.document, keyPath) }));
+
+    const initialLinks = storedDoc.links;
+    let creLinks = initialLinks.filter(
+      (x) => x.document && !keyPath.includes(getStoreKey(x.document)) && getStoreKey(x.document) in dataStore
```

Comment:
This was filtering out the Standards, so I've separated it out and added the standards below.